### PR TITLE
Added dotnet7-sdk

### LINKED ISSUE #1483: [Request]: dotnet7-sdk
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] English interface (or at least English documentation)
- [X] Alternative, forked or prerelease version of an existing package
- [X] Fairly standard install (e.g. no elaborate pre/post install scripts)

### Name

dotnet7-sdk

### Description

.NET is a free, cross-platform, open source developer platform for building many different types of applications.

### Homepage

https://dotnet.microsoft.com/en-us/download/dotnet

### Download Link(s)

https://dotnet.microsoft.com/en-us/download/dotnet/7.0

### Corresponding package

versions/dotnet7-sdk

### Some Indication of Popularity/Repute

_No response_

### PR DESCRIPTION
**Pull request by thedjdoorn**
Since .NET 8 is the current version as of November 14th, version 7 should be included in the versions bucket. This might also be a fix for Scoop-managed applications that still use older .NET versions (i.e. `extras/fancontrol`, which also has a .Net 8 version, but the maintainer didn't bother). It would also be a good idea to consider adding a .NET 7 version of `extras/windowsdesktop-runtime` so users don't need the entire SDK, but that's for a different issue.

This manifest has been copied from the last .NET 7 release on main, with a small version bump to 7.0.404

Closes #1483

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/dotnet7-sdk.json
Comment (val1984) [inline bucket/dotnet7-sdk.json:31]: Changing this line to this should make checkver pass:
```json
"jsonpath": "$..releases-index[?(@.channel-version == '7.0')].latest-sdk",
```

**Review by val1984 — CHANGES_REQUESTED** [review]
Thank you for your PR. As written in the [contributing guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md), it should pass the `/verify` check before being considered for human review by someone who has push rights on the repo (unfortunately not me 🙂).

**Review by val1984 — APPROVED** [review]
Everything looks good, now awaiting a review by someone with push privileges 🙂

**Comment by lsymds** [comment]
Any chance of getting this reviewed @HUMORCE ? Thanks!

**Review by HUMORCE — APPROVED** [review]

**Comment by lsymds** [comment]
Thanks @HUMORCE - appreciate it.

**Pull request merged**
